src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,9 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
```

Comment:
PropTypes was not used to validate the props for the component. It’s a good practice to use PropTypes in order to document the intended types of properties passed to components. This can help to identify potential problems during development.